prboom2/src/e6y_launcher.c

```diff
@@ -655,10 +655,12 @@ static dboolean L_PrepareToLaunch(void)
     index = (int)SendMessage(launcher.listIWAD, CB_GETITEMDATA, index, 0);
     if (index != CB_ERR)
     {
+      extern void D_AutoloadIWadDir();
       char *iwadname = PathFindFileName(launcher.files[index].name);
       history = malloc(strlen(iwadname) + 8);
       strcpy(history, iwadname);
       AddIWAD(launcher.files[index].name);
+      D_AutoloadIWadDir();
```

Comment:
Is this now called twice, i.e. once here from `LauncherShow()` and once earlier in `D_DoomMainSetup()`?

Reply:
Alright, so this takes account of the `wadfiles[]` array being killed and then rebuilt by the launcher, right?

Reply:
Yes. I tested it on Windows, seems to work.